game_board.py

```diff
@@ -106,58 +144,70 @@ def print_choice(screen,selected_move_idx,move):
     screen.refresh()    
 
 
-def move_down(move,current_row):
-    """ Finds the "+" char which has the same xpos but an other Ypos in the move list
-    return the index of the new "+"
+def move_down(plus_list,current_row):
+    """ Finds the "+" char that is below the current_row. If the current "+" char is at the bottom, 
+        it finds the "+" char at the top with the same x position
 
-    current_row -- the currently selected row in the move
-    move -- the lists of all the "+" and their positions
+    current_row -- the currently selected row in the plus_list
+    plus_list -- the lists of all the "+" and their positions
     """
     
     new_row = current_row
     
     while True:
         new_row += 1
-        if new_row == len(move):
+        if new_row == len(plus_list):
             new_row = 0
         
-        if move[current_row][1] == move[new_row][1] and move[current_row][2] != move[new_row][2]:
+        if plus_list[current_row][1] == plus_list[new_row][1] and plus_list[current_row][2] != plus_list[new_row][2]:
      
           return new_row 
 
-def move_up(move,current_row):
-    """ Finds the "+" char which has the same xpos but an other Ypos in the move list
-    return the index of the new "+"
+def move_up(plus_list,current_row):
+    """ Finds the "+" char that is above the current_row. If the current "+" char is at the bottom, 
+        it finds the "+" char at the top with the same x position
+
 
-    current_row -- the currently selected row in the move
-    move -- the lists of all the "+" and their positions
+    current_row -- the currently selected row in the plus_list
+    plus_list -- the lists of all the "+" and their positions
     """
     
     new_row = current_row
     
     while True:
         if new_row == 0:
-            new_row = len(move)
+            new_row = len(plus_list)
         
         new_row -= 1
         
-        if move[current_row][1] == move[new_row][1] and move[current_row][2] !=   move[new_row][2]:
+        if plus_list[current_row][1] == plus_list[new_row][1] and plus_list[current_row][2] !=   plus_list[new_row][2]:
```

Comment:
This if statement is difficult to understand. I would suggest moving each x and y position to separate variables to make it more readable.
Like this:
```
current_x = plus_list[current_row][1]
new_x = plus_list[new_row][1]

if current_x == new_x ...
```